### PR TITLE
Start fixing PHPMD issues

### PR DESCRIPTION
**Pull request by brunoocasali**
Fix part of the phpmd issues (most of the simpler ones) like long variable names, too many lines in the same method, and useless else conditions.

### CODE REVIEW

File: src/Collection.php [hunk 1]
```diff
@@ -4,6 +4,13 @@
 
 namespace Meilisearch\Bundle;
 
+//
```
Comment (bidoubiwa) [inline src/Collection.php:7]:
```suggestion
```

Is this useful?

File: src/Command/MeilisearchImportCommand.php [hunk 2]
```diff
@@ -110,7 +110,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $repository = $manager->getRepository($entityClassName);
             $classMetadata = $manager->getClassMetadata($entityClassName);
             $entityIdentifiers = $classMetadata->getIdentifierFieldNames();
-            $sortByIdentifiersParam = array_combine($entityIdentifiers, array_fill(0, count($entityIdentifiers), 'ASC'));
+            $sortByAttrsParam = array_combine($entityIdentifiers, array_fill(0, count($entityIdentifiers), 'ASC'));
```
Comment (bidoubiwa) [inline src/Command/MeilisearchImportCommand.php:113]: What does an object look like? is `sortByAttrs` not enough?

**Review by bidoubiwa — COMMENTED** [review]
Still a linting issue in your CI tests apparently

File: src/Command/MeilisearchImportCommand.php (hunk 2 above)
Comment (brunoocasali) [inline src/Command/MeilisearchImportCommand.php:113]:
```suggestion
            $sortByAttrs = array_combine($entityIdentifiers, array_fill(0, count($entityIdentifiers), 'ASC'));
```

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Comment by brunoocasali** [comment]
bors merge

**Pull request merged**